src/components/TableData/labelStyle.style.js

```diff
@@ -0,0 +1,12 @@
+export const labelStyle = {
+  color: "#00000050",
+  fontWeight: "400",
+  fontFamily: "'Inter', sans-serif",
+  fontSize: 14,
+}
+export const dataStyle = {
+  color: "#000",
+  fontWeight: "400",
+  fontFamily: "'Inter', sans-serif",
```

Comment:
The font family should be global, so that you can avoid having to write it every time you need it